tests/cases/fourslash/codeFixInferFromUsageContextualImport4.ts

```diff
@@ -0,0 +1,30 @@
+/// <reference path="fourslash.ts" />
+
+// @strict: true
+// @noImplicitAny: true
+// @noLib: true
+
+// @Filename: /types.d.ts
+////declare function getEmail(user: import('./a').User, settings: import('./a').Settings): string;
```

Comment:
this example is weird, in that it reflects how we would do things but not how an npm package would do things. Probably a more realistic test would be:

```ts
// @filename: index.ts
import { User, Settings } from './a'
export function getEmail(user: User, settings: Settings): string
```

It probably tests exactly the same code paths internally, so I'd prefer to see it if you only want to put one test like this in.